src/sparsezoo/objects/recipes.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Dict, List, Union
+
+from sparsezoo.objects import File
+
+
+__all__ = ["Recipes"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Recipes:
+    """
+    Object to store a list of recipes for a downloaded model and pull the default
+
+    :param recipes: list of recipes to store
+    :param stub_params: dictionary that may contain custom default recipes names
+    """
+
+    _RECIPE_DEFAULT_NAME = "recipe.md"
+
+    def __init__(
+        self, recipes: Union[None, File, List[File]], stub_params: Dict[str, str]
```

Comment:
Union[None,...] is same as Optional[Union[F, L[F]] = None
and i would make stub_params: Dict[str, str] = {}